tests/listsAndDropDowns.spec.ts

```diff
@@ -10,61 +10,50 @@ test.describe('Lists and Dropdowns', async () => {
 
 	test('Validate selected pet types from the list', async ({ page }) => {
 		const ownerFullName = 'George Franklin'
+		const petTypeField = page.locator('#type1')
 
-		await page.getByText(ownerFullName).click()
+		await page.getByRole('link', { name: ownerFullName }).click()
 		await expect(page.locator('.ownerFullName')).toHaveText(ownerFullName)
 
-		await page.getByText('Edit pet').click()
+		await page.getByText('Leo').locator('..').getByRole('button', { name: 'Edit Pet' }).click()
```

Comment:
This is better, but try to avoid using .locator('..'). There is a better way rather than going one level up in the DOM.
`await page.locator("app-pet-list", { hasText: "Leo" }).getByRole("button", { name: "Edit Pet" }).click();`